mrmustard/settings.py

```diff
@@ -269,6 +270,18 @@ def SEED(self, value):
         self._seed = value
         self.rng = np.random.default_rng(self._seed)
 
+    @property
+    def PRECISION_BITS_HERMITE_POLY(self):
+        r"""
+        The number of bits used to represent a single Fock amplitude when calculating Hermite polynomials.
+        Default is 128 (i.e. the Fock representation has dtype complex128).
+        """
+        return self._precision_bits_hermite_poly
+
+    @PRECISION_BITS_HERMITE_POLY.setter
+    def PRECISION_BITS_HERMITE_POLY(self, value: str):
+        self._precision_bits_hermite_poly = value
```

Comment:
Thanks :) 
One final suggestion: Could you check that the value is valid, and error if it's not?

Reply:
At the moment it's checked in `mrmustard.math.tensorflow_wrapper.hermite_renormalized`. I think it might be better not to define a list of allowed values in multiple places (as the list will get longer in my next PR). What do you think?

Reply:
I agree!
Slight preference for moving the check here (as a user, I would rather knowing that I made a mistake right away, as opposed to running some code and then getting an error), but don't waste too much time. The important thing is that there is a check in place

Reply:
I would move it here too, if anything it would be run just once as opposed at every call of `hermite_renormalized`